Snippets/Sqs/Sqs_All/SqsTransportConfigurationExtensions.cs

```diff
@@ -33,7 +33,9 @@ public static void ConfigureSqsTransport(this TransportExtensions<SqsTransport>
             var validValue = bool.TryParse(nativeDeferralRaw, out var nativeDeferral);
             if (validValue && nativeDeferral)
             {
+#pragma warning disable 618
```

Comment:
The API never really had an effect. Should we remove it?

Reply:
👍